docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  web:
+    build:
+      context: .
+    depends_on:
+      - db
+    links:
+      - db
+    volumes:
+      - .:/app
+    ports:
+      - 4000:4000
+    environment:
+      - DATABASE=postgres
+      - DATABASE_USERNAME=postgres
+      - DATABASE_PASSWORD=mysecretpassword
+      - DB_HOST=db
+      - DB_DIALECT=postgres
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=mysecretpassword
+      - POSTGRES_DB=postgres
```

Comment:
Yes, that's correct, they are used by the `postgres` official docker image. See **Environment Variables** section of https://hub.docker.com/_/postgres/. Over time, the values will be removed from the `docker-compose.yml` file, particularly if we deploy somewhere, they'll likely need to be injected from somewhere, but for running locally this works fine for now.